microsetta_interface/static/vendor/emperor/templates/logic-template.html

```diff
@@ -0,0 +1,155 @@
+<div id='{{ plot_id }}' style="position: relative; width:{{ width }}; height:{{ height }};">
```

Comment:
You don't need this file.

Reply:
I think the messiest part about maintaining this integration is figuring out exactly what our api to emperor should be.  I'm worried that if updating to the newest version of emperor is any more complex than unzipping a file and drag dropping it into /static/vendor/emperor, then we're either going to run into constant problems on updates or we're going to be stuck using old versions for long periods of time.  

It seems we have a script that is capable of generating such a zip file (save for the changes I made to split index.html, which I'd also like to offload somehow), if we're worried about keeping this file and the other templates you noted then we should remove them as part of the configuration of the script that builds our installs?

Reply:
Having a small script that can run the import would probably not be a bad idea. This would likely be not too bad to handle using the Emperor Python API. This way dependencies are handled via pip/conda.